dbos/dbos.py

```diff
@@ -138,16 +171,64 @@ def send_temp_workflow(
         temp_send_wf = _workflow_wrapper(self, send_temp_workflow)
         set_dbos_func_name(send_temp_workflow, TEMP_SEND_WF_NAME)
         self._register_wf_function(TEMP_SEND_WF_NAME, temp_send_wf)
+
+    @property
+    def sys_db(self) -> SystemDatabase:
+        if self._sys_db is None:
```

Comment:
What's going on here? When does this happen?

Reply:
Design changed since this conversation